app/_common/api/profile.ts

```diff
@@ -51,3 +51,30 @@ export const getReceiveLikeCount = async (
     }
   }
 };
+
+interface ProfileLikeBody {
+  senderId: number;
+  receiverId: number;
+}
+
+interface ProfileLikeResponse {
+  id: number;
```

Comment:
```suggestion
  id: Creator["id];
```
유지보수에 도움이 될 것 같아서 제안드려요!

Reply:
오!! 배워갑니다 👍 👍 👍